packages/aws-cdk-lib/aws-ses/lib/dedicated-ip-pool.ts

```diff
@@ -19,11 +36,22 @@ export interface IDedicatedIpPool extends IResource {
  */
 export interface DedicatedIpPoolProps {
   /**
-   * A name for the dedicated IP pool
+   * A name for the dedicated IP pool.
+   * 64 character limit. Only lower case letters, numbers, periods, underscores, and dashes are permitted.
```

Comment:
```suggestion
```
We should open a separate PR that adds validation on `dedicatedIpPoolName` (I think it should be a `chore`).
The validation pattern is specified [here](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ses-dedicatedippool.html#cfn-ses-dedicatedippool-poolname).
You're welcome to submit further contributions if you want to go for it 💪

Reply:
I'll try it later!